### DISCUSSION TITLE
Add ability to check in "Groups" of people at once

### DISCUSSION THREAD

**Discussion by ChillLP** [Traewelling/traewelling#2053] [Ideas / Feature Requests]
General idea is that it works like a Whatsapp group, you invite people into that group and for the set timeframe every check-in will be Mirrored for the others.
this would make träwelling in groups a whole lot easier and you wouldnt have to send each user a new notification (maybe send them one saying "Your group host checked in" or something along the lines) that the same people are on that specific train ride, rather send others on that ride a message saying "Group xyz is also on your ride" and when clicking opening that notification they can see who is in that group and where they are headed. 

![image](https://user-images.githubusercontent.com/105374102/212931911-ba3f7b24-a68c-4dd2-9096-5239587ae1d4.png)

**Comment by pascalwittler** [comment]
Personally, I'm not a fan of

* the concept of having your account's check-in data changed (or added) by someone else (a group leader or another member?),
* additional steps like clicking on a group name to get information about who you are meeting on the train (each person),
* being part of (multiple?) groups for an indefinite period of time that may be doing check-ins for trips I'm not on every time and/or
* having the same potential problems as with WhatsApp groups (group admins with special rights, endless notifications, etc.).

Probably the current behavior of single notifications per user checking in to your train could be improved by creating a comma-separated list (e.g. „A, B and C are also on your ride“) and/or (my personal preference) by implementing #2011 instead.

**Comment by MrKrisKrisu** [comment]
It is now possible to check in your friends.

See [Release v1.79 (api)](https://github.com/Traewelling/traewelling/releases/tag/v1.79) and [Release v1.82 (ui)](https://github.com/Traewelling/traewelling/releases/tag/v1.82)
